client/src/Question.svelte

```diff
@@ -50,13 +50,13 @@
 	}
 
 	async function toggle(what) {
-		await fetch(`/api/event/${event.id}/questions/${event.secret}/${question.qid}/toggle/${what}`, {
+		const res = await fetch(`/api/event/${event.id}/questions/${event.secret}/${question.qid}/toggle/${what}`, {
 			"method": "POST",
 			"body": question[what] ? "off" : "on",
-		});
+		}).then(r => r.json());
 		localAdjustments.update(la => {
 			let q = la.remap[question.qid] || {};
-			q[what] = !question[what];
+                        q[what] = res[what];
```

Comment:
I'm not sure if this will work when `answered` is _unset_, at least if that means that the value is unset in the JSON from the server. It'll mean that `q[what]` gets set to `null` (or `undefined`?), rather than getting removed. That is arguably what we want, but I don't know if the code that _applies_ `localAdjustments` does the right thing if it sees an adjustment that says `["answered"] = null`.

Reply:
Yea, this code is relying pretty heavily on Javascript semantics, and takes advantage of the fact that an explicit `undefined` and a missing property will both resolve to undefined (example below): 
```javascript 
let q = {  answered: undefined };
let adj = {}
q.answered === adj.answered // true
```
But the explicit set will still resolve to true in the  following code `"answered" in adj`, so the adjustment code will still run where needed.

In practice I think this still works as expected, but I agree that it's not very clear what is going on. I'll see if I can find a better solution tomorrow.